packages/deploy-tooling/src/base/deploy.ts

```diff
@@ -345,28 +345,21 @@ export async function createTransportRequest(
     logger: Logger,
     provider?: AbapServiceProvider
 ): Promise<string> {
-    try {
-        if (!provider) {
-            provider = await getAbapServiceProvider(config, logger);
-        }
-        const adtService = await provider.getAdtService<TransportRequestService>(TransportRequestService);
-        logger.debug(`ADTService created for application ${config.app.name}, ${!adtService}.`);
-        const transportRequest = await adtService?.createTransportRequest({
-            packageName: config.app.package ?? '',
-            ui5AppName: config.app.name,
-            description: 'Created by @sap-ux/deploy-tooling'
-        });
-        if (transportRequest) {
-            logger.info(`Transport request ${transportRequest} created for application ${config.app.name}.`);
-            return transportRequest;
-        }
-        throw new Error(`Transport request could not be created for application ${config.app.name}.`);
-    } catch (e) {
-        logger.error(
-            `Transport request could not be created for application ${config.app.name}. Please create it manually and re-run deployment configuration for this project.`
-        );
-        throw e;
+    if (!provider) {
+        provider = await getAbapServiceProvider(config, logger);
+    }
+    const adtService = await provider.getAdtService<TransportRequestService>(TransportRequestService);
+    logger.debug(`ADTService created for application ${config.app.name}: ${!!adtService}.`);
```

Comment:
Not sure if this logger is actually needed, from playing with this functinality over the past week, the logger.info or the new Error should provide everything you need.

Reply:
Removed